src/dialect/keywords.rs

```diff
@@ -422,7 +422,12 @@ pub const RESERVED_FOR_TABLE_ALIAS: &[&str] = &[
     // Reserved as both a table and a column alias:
     WITH, SELECT, WHERE, GROUP, HAVING, ORDER, LIMIT, OFFSET, FETCH, UNION, EXCEPT, INTERSECT,
     // Reserved only as a table alias in the `FROM`/`JOIN` clauses:
-    ON, JOIN, INNER, CROSS, FULL, LEFT, RIGHT, NATURAL, USING,
+    ON, JOIN, INNER, CROSS, FULL, LEFT, RIGHT, NATURAL, USING, LIMIT, OFFSET, FETCH,
```

Comment:
`, LIMIT, OFFSET, FETCH,` here is the result of a rebase going bad..

Reply:
I'll fix this in a moment.

Reply:
D'oh! Sorry. I distinctly remember intentionally resolving this conflict in favor of "adding" the new keywords without realizing they were in fact duplicates.